src/Modal.tsx

```diff
@@ -478,7 +485,9 @@ const Modal: Modal = (React.forwardRef(
           onMouseDown={handleDialogMouseDown}
           className={dialogClassName}
         >
-          {children}
+          <div className={classNames(`${bsPrefix}-content`, contentClassName)}>
```

Comment:
**issue:** why move this here instead of just passing `contentClassName` down to `<ModalDialog>`? this is technically a breaking change in the API

Reply:
Honestly, because Im new to this and am not really familiar! Just want to see this made, what do I need to do?

Reply:
I also thought it made sense to go here since backdropClassName and dialogClassName both exist on Modal

Reply:
pass down `dialogClassName` from the modal component to the dialog component, and keep rendering that `<div>` inside the dialog component

also, it'd be nice to see a test for this

Reply:
Updated.

> it'd be nice to see a test for this

If you have an example of what you're looking for, I'll get you what you need.

Reply:
yup, take a look at https://github.com/react-bootstrap/react-bootstrap/blob/f846dfbe9b8d85fa9ec8cd3d7e50d1310fd12220/test/ModalSpec.js#L192-L199

Reply:
Thanks, added